src/pudl/extract/excel.py

```diff
@@ -208,7 +208,7 @@ def extract(self, **partitions):
             if page in self.BLACKLISTED_PAGES:
                 logger.debug(f'Skipping blacklisted page {page}.')
                 continue
-            df = pd.DataFrame()
+            dfs = [pd.DataFrame(), ]
```

Comment:
Does this list need an empty dataframe?

Reply:
Sadly, it does need an empty dataframe, because there are cases in which the following loop is never entered and nothing gets added to the list, and the subsequent call to `pd.concat()` fails. I'm sure this could be better.